debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/sink/WriteModelStrategyHelper.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.mongodb.sink;
+
+import org.apache.kafka.connect.errors.DataException;
+import org.bson.BsonDocument;
+
+import com.mongodb.client.model.WriteModel;
+
+import io.debezium.connector.mongodb.converters.sink.SinkDocument;
+
+public final class WriteModelStrategyHelper {
```

Comment:
Is this class even used?

Reply:
I refactored this part. It's not in use anymore.